src/Command/QueueTasksCommand.php

```diff
@@ -157,9 +157,10 @@ private function formatTaskLabel (Task $task) : string
 		if (null !== $task->group)
 		{
 			return \sprintf(
-				"<fg=blue>%s</>: %s",
+				"<fg=blue>%s</>: %s (%s)",
```

Comment:
We talked about it, but can you look up how I render the key in the list and use the same styling (color) here then?